newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -774,7 +774,9 @@ export default class ThemableEventsTree extends Component<
                   this.props.onAddInstructionContextMenu(event, ...args)
                 }
                 onOpenExternalEvents={this.props.onOpenExternalEvents}
-                onOpenLayout={this.props.onOpenLayout}
+                onOpenLayout={(name: string) => {
+                  this.props.onOpenLayout(name);
+                }}
```

Comment:
It's needed to comply with flow since now onOpenLayout returns the editorsTab but the props does not allow to return something. I didn't want to change all the types for all the components to which this props is drilled to